### PR TITLE
Stream product collection task 1

### PR DESCRIPTION
**Pull request by BulavaKatsyaryna**

### CODE REVIEW

File: JiS7.iml [hunk 1]
```diff
@@ -11,5 +11,14 @@
     </content>
```
Comment (KaydunovDenis) [inline JiS7.iml:11]: This file does not need to be pushed to GitHub.

File: .idea/.gitignore [hunk 2]
```diff
@@ -1,74 +0,0 @@
...
-
-# Mongo Explorer plugin
-.idea/**/mongoSettings.xml
-
-# File-based project format
-*.iws
-
-# IntelliJ
-out/
-
```
Comment (KaydunovDenis) [inline .idea/.gitignore:54]: Write here:
*.iml

File: src/main/java/LECTURE_Stream_API_And_Lambdas/Task3_Product1/Demo.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Demo {
+
+    public static void main(String[] args) {
+
+
```
Comment (KaydunovDenis) [inline src/main/java/LECTURE_Stream_API_And_Lambdas/Task3_Product1/Demo.java:11]: Delete empty lines

File: src/main/java/LECTURE_Stream_API_And_Lambdas/Task3_Product1/Demo.java [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+        listProduct.add(product);
+        listProduct.add(product2);
+        listProduct.add(product3);
+        listProduct.add(product4);
+        listProduct.add(product5);
+        listProduct.add(product6);
+        listProduct.add(product7);
+        listProduct.add(product8);
+
```
Comment (KaydunovDenis) [inline src/main/java/LECTURE_Stream_API_And_Lambdas/Task3_Product1/Demo.java:30]: If you use java version 11+, then you can use this construction:
 Product product = new Product("BubbleGum", 2.50, 2.0);
        Product product2 = new Product("Lays", 3.20, 50.0);
        Product product3 = new Product("Milk", 1.50, 0.0);
.... и т.д.
List<Product> listProduct = new ArrayList<>(){
add(product);
add(product1);
add(product)2;
.... и т.д.
}

File: src/main/java/LECTURE_Stream_API_And_Lambdas/Task3_Product1/Product.java [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class Product {
+
```
Comment (KaydunovDenis) [inline src/main/java/LECTURE_Stream_API_And_Lambdas/Task3_Product1/Product.java:11]: empty line

**Review by KaydunovDenis — COMMENTED** [review]
Read my comment.

File: JiS7.iml [hunk 6]
```diff
@@ -11,5 +11,14 @@
...
     <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="module-library">
+      <library>
+        <CLASSES>
+          <root url="jar://$MAVEN_REPOSITORY$/org/projectlombok/lombok/1.18.18/lombok-1.18.18.jar!/" />
+        </CLASSES>
+        <JAVADOC />
+        <SOURCES />
+      </library>
+    </orderEntry>
```
Comment (ArtikTico) [inline JiS7.iml:22]: гит игнор Каать

File: src/main/java/LECTURE_Stream_API_And_Lambdas/Task3_Product1/Demo.java (hunk 4 above)
Comment (ArtikTico) [reply src/main/java/LECTURE_Stream_API_And_Lambdas/Task3_Product1/Demo.java:30]: А еще лучше List.of(product1,product2 ...)

**Review by ArtikTico — COMMENTED** [review]
Чекни наши комменты исправь и вмержим

**Pull request closed without merging**